scripts/create_runtime_policy.sh

```diff
@@ -221,25 +252,30 @@ then
         elif [[ -x "/usr/lib/dracut/skipcpio" ]] ; then
             /usr/lib/dracut/skipcpio $i | gunzip -c | cpio -i -d 2> /dev/null
         else
-            echo "ERROR: No tools for initramfs image processing found!"
-            break
+            announce "ERROR: No tools for initramfs image processing found!"
+            exit 1
         fi
 
-        find -type f -exec $ALGO "./{}" \; | sed "s| \./\./| /|" >> ${OUTPUT}_allowlist
+        find -type f -exec $ALGO "./{}" \; | sed "s| \./\./| /|" >> $ALLOWLIST_DIR/${OUTPUT}
     done
 fi
 
 # A bit of cleanup on the resulting file (among other problems, sha256sum might output a hash with the prefix '//')
-sed -i "s^ //^ /^g"  ${OUTPUT}_allowlist 
-sed -i 's^"\\\\^"^g' ${OUTPUT}_allowlist
+sed -i "s^ //^ /^g"  $ALLOWLIST_DIR/${OUTPUT} 
+sed -i "s/^\\\//g" $ALLOWLIST_DIR/${OUTPUT}
```

Comment:
So this is removing a leading `\/` ? Mabe add to documentation above.

Reply:
Just added a better explanation to both